shell/platform/android/test/io/flutter/plugin/platform/PlatformViewWrapperTest.java

```diff
@@ -117,7 +117,6 @@ public void draw(Canvas canvas) {
     verify(surface, times(1)).lockHardwareCanvas();
     verify(surface, times(1)).unlockCanvasAndPost(canvas);
     verifyNoMoreInteractions(surface);
-    verifyNoMoreInteractions(canvas);
```

Comment:
No longer expected due to https://github.com/robolectric/robolectric/pull/7279; there are now calls to `draw` that occur on `canvas`.